.github/workflows/build.yaml

```diff
@@ -1,49 +1,43 @@
-name: Update devs api image
+name: Build, tag, and publish
 env:
   # This must be defined for the bash redirection
   GOOGLE_APPLICATION_CREDENTIALS: 'jade-dev-account.json'
   # This must be defined for the bash redirection
   GOOGLE_SA_CERT: 'jade-dev-account.pem'
 on:
   workflow_dispatch: {}
+  pull_request:
```

Comment:
If this is run on PRs, should it be added to the set of required checks on PRs?

Reply:
Yes, I think so, but that should happen after the PR is merged.